pkg/hypervisor/hypervisor.go

```diff
@@ -54,13 +54,14 @@ type VisorConn struct {
 // Hypervisor manages visors.
 type Hypervisor struct {
 	c      Config
+	assets http.FileSystem             // Web UI.
 	visors map[cipher.PubKey]VisorConn // connected remote visors.
 	users  *UserManager
 	mu     *sync.RWMutex
 }
 
 // New creates a new Hypervisor.
-func New(config Config) (*Hypervisor, error) {
+func New(assets http.FileSystem, config Config) (*Hypervisor, error) {
```

Comment:
What do you think of putting `assets` as the second argument? In my opinion, `config` always should be the first.

Reply:
@nkryuchkov I've always had the habit of putting `config` as the last argument. 😅